### DISCUSSION TITLE
Capabilities not found error after codemagic build

### DISCUSSION THREAD

**Discussion by FrixQn** [codemagic-ci-cd/codemagic-docs#2544] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

- Unity iOS Workflow

### Steps to reproduce

Too much to descibe this

### Expected results

Too much to descibe this

### Actual results

Hi! I use OneSignal extension in my Unity and Flutter projects, on bought projects after build and release ipa on testflight i have an error with text: 
 'Push Notifications' capability missing! Add the capability in Xcode under 'Target' -> '<MyAppName(MainTarget)>' -> 'Signing & Capabilities' then click the '+ Capability' button.
at the same time i create in Unity with Build.BuildIos method ios folder, and there are PushNotif capability (attach screenshoot) 
And in my IOS dev account i have this capabilities and app group too 
When i build project from xcode everything fine, but after build codemagic i have error
On flutter after build with default workflow UI i have same error
I need to build apps usnig codemagic with fix of adding capabilities, how i can fix that ?

### Build id (optional)

664ad8dca9826554ae54f955

**Comment by dtrdic** [comment]
hey @FrixQn i can see a different error for this build you shared, but nothing related to missing capabilities.
_`The bundle version must be higher than the previously uploaded version`_
Which typically occurs because you've previously uploaded an app version with the same bundle version to the App Store.
Furthermore i can see in the `Set up code signing identities` step that the provisioning profile with the OneSignalExtension is also imported, so everything seems to be all right.
Can you please double-check the build id you shared, maybe I'm looking at the wrong build

**Comment by dtrdic** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
